lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -311,6 +311,15 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       extractVersion: '^php-(?<version>\\S+)',
     },
   },
+  pnpm: {
+    asdfPluginUrl: 'https://github.com/jonathanmorley/asdf-pnpm',
+    config: {
+      datasource: GithubReleasesDatasource.id,
+      packageName: 'pnpm/pnpm',
+      versioning: semverVersioning.id,
```

Comment:
remove versioning, it will then use default npm versioning from datasource

Reply:
Removing the line is breaking the code. Could you please provide a code snippet of the change you are suggesting?